src/overlap_check/test/overlap_test_driver.cc

```diff
@@ -0,0 +1,30 @@
+#include <string>
+#include <stdio.h>
+
+#include <gtest/gtest.h>
+
+int main(int argc, char* argv[]) {
+  for (int i = 0; i < argc; i++) {
+    std::string arg = argv[i];
+    if (arg == "--help") {
```

Comment:
Why did I think that `gtest` automatically generated its own help message?

Reply:
It does! That's what I get for copy-pasting from other drivers without question.

Reply:
I've removed it from the overlap test, but I'll leave other test drivers to be updated as a group in a separate PR. Any known reason for this set of code reappearing in our drivers? @ljacobson64? @makeclean?